DirectProgramming/C++/CompilerInfrastructure/guided_matmul_opt_report/sample.json

```diff
@@ -0,0 +1,44 @@
+{
+  "guid": "D61F9365-9455-4726-A272-B566618D7B5F",
+  "name": "Matrix Multiply",
+  "categories": ["Toolkit/oneAPI Direct Programming/C++/Compiler Infrastructure"],
+  "description": "Demonstrates how compiler auto-vectorization can improve the performance of a program. The optimization report option is used to identify potential points of performance improvement.",
+  "toolchain": ["dpcpp"],
+  "os": ["linux"],
+  "targetDevice": ["CPU"],
+  "builder": ["make"],
+  "languages": [{"cpp":{}}],
+  "ciTests": {
+    "linux": [
+      {
+        "id": "baseline",
+        "steps": [
+          "make clean",
+          "make baseline"
```

Comment:
@krisdale If you want the CI to run the sample after running make (to verify that it works), I would add ``./MatVector``